Resources/Prototypes/Entities/Objects/Specific/Janitorial/spray.yml

```
@@ -99,6 +99,36 @@
   - type: Tag
     tags:
       - Spray
+
+- type: entity
+  parent: SprayBottle
+  id: BorgSprayBottle
+  name: internal spray jet
+  suffix: Filled
+  description: A pressurized vessel for spraying reagents, installed directly into a custodial cyborg. Typically filled with space cleaner for dealing with those nasty spills.
+  components:
+  - type: SolutionContainerManager
+    solutions:
+      spray:
+        maxVol: 200
+        reagents:
+        - ReagentId: SpaceCleaner
+          Quantity: 200
+
+- type: entity
+  parent: MegaSprayBottle
+  id: BorgMegaSprayBottle
+  name: adv internal spray jet
```

try not to use abbreviations
```suggestion
  name: advanced internal spray jet
```

---

The name is too long to be visible if its not abbreviated.

---

Oh you're referring to the hand slot thing. I mean... people know what it is. I'd say it's unrelated.

---

I guess... yeah ill change that back.